tests/ClientTest.php

```diff
@@ -46,6 +46,64 @@ public function testGetSupportedEndpoints()
         $this->assertContains('wikipediaSearch', $endpoints);
     }
 
+    public function testGetLastTotalResultsCount()
+    {
+
+        $g =& $this->client;
+
+        // make sure we start from scratch
+        $this->setUp();
+
+        // check for current value
+        $total = $g->getLastTotalResultsCount();
+
+        // make sure it's currently null
+        $this->assertNull($total);
+
+        // search for a large result
+        $arr = $g->search([
+            'q'    => '東京都',
+            'lang' => 'en',
+        ]);
+
+        $total = $g->getLastTotalResultsCount();
+
+        $this->assertIsInt($total);
+        $this->assertGreaterThan(100, $total);
+
+        // search for a couple results
+        $arr = $g->search([
+            'name_equals'  => 'Grüningen (Stedtli)',
+            'country'      => 'CH',
+            'featureClass' => 'P',
+        ]);
+
+        $this->assertIsArray($arr);
+        $this->assertNotEmpty($arr);
+
+        $count = count($arr);
+
+        $total = $g->getLastTotalResultsCount();
+
+        $this->assertIsInt($total);
+        $this->assertEquals($count, $total);
+
+        // search for a non-existing place
+        $arr = $g->search([
+            'name_equals'    => 'öalkdjfpaoirhpauhrpgjanfdlijgbiopesrzgpi'
+        ]);
+
+        $this->assertIsArray($arr);
+        $this->assertEmpty($arr);
+
+        $total = $g->getLastTotalResultsCount();
+
+        $this->assertIsInt($total);
+        $this->assertEquals(0, $total);
+
```

Comment:
Please remove unneeded line breaks. I'll see why PHPCS isn't catching up on this automatically 😵